parcels/compilation/codegenerator.py

```diff
@@ -624,7 +624,7 @@ def visit_Call(self, node):
                     else:
                         node.ccode = rhs
             except:
-                raise RuntimeError("Error in converting Kernel to C. See https://nbviewer.jupyter.org/github/OceanParcels/parcels/blob/master/parcels/examples/tutorial_parcels_structure.ipynb#3.-Kernel-execution for hints and tips")
+                raise RuntimeError("Error in converting Kernel to C. See https://nbviewer.jupyter.org/github/OceanParcels/parcels/blob/master/docs/examples/tutorial_parcels_structure.ipynb#3.-Kernel-execution for hints and tips")
```

Comment:
Link to the docs-version (https://docs.oceanparcels.org/en/latest/examples/tutorial_parcels_structure.ipynb#3.-Kernel-execution)?